reaktive/src/jvmNativeCommonTest/kotlin/com/badoo/reaktive/utils/ThreadLocalStorageThreadingTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.badoo.reaktive.utils
+
+import com.badoo.reaktive.scheduler.computationScheduler
+import com.badoo.reaktive.utils.atomic.AtomicReference
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class ThreadLocalStorageThreadingTest {
```

Comment:
Merge with `ThreadLocalStorageTest`. They previously were testing different classes, but now it is only one.